Package.swift

```diff
@@ -224,8 +227,7 @@ let package = Package(
                 .product(name: "OpenAPIRuntime", package: "swift-openapi-runtime"),
                 .product(name: "Logging", package: "swift-log"),
                 .product(name: "Atomics", package: "swift-atomics"),
-                .product(name: "Crypto", package: "swift-crypto"),
-                .product(name: "LeafKit", package: "leaf-kit")
+                .product(name: "Crypto", package: "swift-crypto")
```

Comment:
```suggestion
                .product(name: "Crypto", package: "swift-crypto"),
```
Minor not - I prefer trailing commas in arrays to avoid this coming up as a change

Reply:
To be clear, haven't been doing it because if i wanted to do it i would have also liked to do it in a complete way and not half-way there.
Also swift-format has been on my to-do list which will solve the problem.

Reply:
Yep very happy to, ideally we'll roll it out across the org

Reply:
sure. I'll try to see what Apple folks use. Then try on this repo, and eventually roll out everywhere.
When we want to roll out, we'll need to set up the config file in the repo, and @gwynne can also update the reusable CI.